workspace/__lib__/xod/graphics/line-colored/patch.xodp

```diff
@@ -0,0 +1,85 @@
+{
+  "description": "Represents a line of the custom style.\n\nThe line is defined in the Cartesian system by two points with coordinates (X0, Y,0) and (X1, Y1).\n\nThe CLR pin value sets the new stroke color of the line.",
+  "nodes": [
+    {
+      "id": "B1HW4IvnW8",
+      "label": "Y0",
+      "position": {
+        "units": "slots",
+        "x": 3,
+        "y": 0
+      },
+      "type": "xod/patch-nodes/input-number"
+    },
+    {
+      "id": "Bkl-NLPnZI",
+      "label": "Y1",
+      "position": {
+        "units": "slots",
+        "x": 5,
+        "y": 0
+      },
+      "type": "xod/patch-nodes/input-number"
+    },
+    {
+      "id": "S14b4Iwh-8",
+      "label": "GFX'",
+      "position": {
+        "units": "slots",
+        "x": 0,
+        "y": 2
+      },
+      "type": "@/output-graphics"
+    },
+    {
+      "id": "SkG-VIwhZU",
+      "position": {
+        "units": "slots",
+        "x": 0,
+        "y": 1
+      },
+      "type": "xod/patch-nodes/not-implemented-in-xod"
+    },
+    {
+      "id": "SkWZEIP2WL",
+      "label": "X0",
+      "position": {
+        "units": "slots",
+        "x": 2,
+        "y": 0
+      },
+      "type": "xod/patch-nodes/input-number"
+    },
+    {
+      "id": "SkXbELD3bI",
+      "label": "GFX",
+      "position": {
+        "units": "slots",
+        "x": 0,
+        "y": 0
+      },
+      "type": "@/input-graphics"
+    },
+    {
+      "description": "Stroke color.",
+      "id": "r1qVIP3b8",
+      "label": "CLR",
```

Comment:
A bit unfortunate label. `CLR` is for CLEAR usually. I’d recommend calling it `C`.